src/theme/components/structuredContent.ts

```diff
@@ -0,0 +1,29 @@
+const StructuredContent = {
+  parts: ["image"],
+  baseStyle: ({ imagePosition }) => {
+    const styles = {};
+    if (imagePosition === "center") {
+      (styles as any).maxWidth = "100%";
+    }
+    return {
+      image: {
```

Comment:
Add `marginBottom: m`.  Currently, the image and captions can be pressed right up against text that runs below the image.